### DISCUSSION TITLE
Python Sul 2022 - Venda de ingressos

### DISCUSSION THREAD

**Discussion by franciscoalface** [apyb/comunidade#25] [Geral]
Bom dia pessoal,

Eu gostaria de saber se a APyB possui uma conta no Sympla ou alguma plataforma que não seja o EventBrite, que nós poderíamos utilizar para a venda de ingressos da Python Sul 2022.
Tivemos uma experiência ruim com o EventBrite quando tivemos que cancelar o evento em 2020 e se for possível usar a APyB para receber diretamente o dinheiro dos ingresso usando outra plataforma de venda de ingressos, seria excelente. :smile:

**Comment by alynnefs** [comment]
@franciscoalface a gente usou a eventbrite na Python Brasil do ano passado pra facilitar a integração com o discord. Não sei como ficou a questão da Python Sul, mas vocês receberam o dinheiro para conseguir reembolsar? porque acredito que não fui reembolsada, mas pode ser que tenha se "perdido" no meio das transações

**Reply by franciscoalface** [reply]
@alynnefs não conseguimos fazer o reembolso pela plataforma. A empresa saiu do Brasil e não conseguimos suporte para resolver o problema. Só conseguimos que o dinheiro fosse enviado para a APyB mais ou menos um ano e meio depois.
Tivemos dois reembolsos feitos somente, que eu fiz do meu bolso e guardei os comprovantes para acertar isso depois.
Nós temos uma reunião da organização na próxima semana e devemos enviar e-mails para todos que compraram ingresso e vamos reembolsar os que assim quiserem. Mas teremos que ver como fazer isso com a APyB, para devolver individualmente os valores.

**Reply by rougeth** [reply]
> Mas teremos que ver como fazer isso com a APyB, para devolver individualmente os valores.

Pergunta que enviei no e-mail:

Sobre os patrocínios e ingressos, vocês tem os dados do dinheiro que entrou e saiu no lado de vocês? Estou perguntando porque essa informação tem que bater com o que temos na APyB.

**Reply by franciscoalface** [reply]
O que temos de valores são R$ 7.844,80. São valores de três patrocínios e o valor dos ingressos vendidos. Este valor bate com o que foi recebido pela APyB? Posso detalhar os valores se for o caso.

**Reply by rougeth** [reply]
Detalha sim, por favor. No lado da APyB, vamos conferir por transação.

**Reply by franciscoalface** [reply]
O que nós temos como recebimentos aqui são essas transações:
Globo | Patrocínio Bronze | 1,000.00
Python Pro | Patrocínio Bronze | 1,000.00
Magrathea Labs | Patrocínio Prata | 2,910.00
Ingressos | EventBrite | 2,934.80

**Reply by franciscoalface** [reply]
@rougeth sabe dizer quem é a pessoa responsável pelo financeiro? Queria confirmar os valores

**Reply by rougeth** [reply]
Pingando o resto da @apyb/diretoria. Eu ainda não tenho acesso a conta, mas quem tiver poderá confirmar os valores.

**Reply by franciscoalface** [reply]
> O que nós temos como recebimentos aqui são essas transações:
> Globo | Patrocínio Bronze | 1,000.00
> Python Pro | Patrocínio Bronze | 1,000.00
> Magrathea Labs | Patrocínio Prata | 2,910.00
> Ingressos | EventBrite | 2,934.80

@muriloviana, você consegue me ajudar a confirmar esses valores?

**Reply by muriloviana** [reply]
Olá @franciscoalface, desculpa a demora, tive alguns problemas com o acesso a conta bancária da associação rs, mas tá tudo certo agora.

Sobre os valores que você levantou, fiz uma busca nos extratos da conta da APyB e temos alguns pontos a serem observados em conjunto 😉:
 
- Globo: o valor recebido na conta da associação foi de R$950.
- Python Pro: tudo ok ✅
- Magrathea Labs: o valor recebido confere, porém a nota fiscal que emitimos para eles foi de R$3.000,00.
- Eventbrite: tudo ok ✅ 
- ICRO: na época foi emitida uma nota fiscal para eles no valor de R$300,00, porém nenhum recebimento foi encontrado nos extratos.

Sabe me dizer o motivo desses valore, Globo e Magrathea, não baterem? E por que a nota fiscal emitida para ICRO não foi paga?

Você pode encontrar mais detalhes nessa issue: https://github.com/apyb/tarefas/issues/78#issuecomment-1156635325

**Reply by franciscoalface** [reply]
O pessoal da Magrathea citou ter que reter 3% de ISS, por isso a diferença de valor. 

Segue o print do e-mail que eles enviaram à época. 

![Screenshot_2022-06-15-13-28-48-00_e307a3f9df9f380ebaf106e1dc980bb6.jpg](https://user-images.githubusercontent.com/3975665/173878688-704a4ea3-7117-4250-b301-36f374941bec.jpg)

Em relação a Globo, não sei dizer. Não tenho nenhuma informação por parte deles sobre essa diferença de valores. Pode ser alguma retenção também, eu imagino.

Em relação a ICRO, eles acabaram enviando o dinheiro em espécie para mim, num envelope.

**Reply by muriloviana** [reply]
Então tudo ok! Apenas o valor da Globo que seria interessante considerar os R$50 reais a menos no orçamento de vocês.

**Comment by rougeth** [comment]
@franciscoalface obrigado por trazer a conversa para o Github :)

> Eu gostaria de saber se a APyB possui uma conta no Sympla ou alguma plataforma que não seja o EventBrite, que nós poderíamos utilizar para a venda de ingressos da Python Sul 2022.

A APyB em si não cria conta nos serviços usados pelo evento (com algumas poucas exceções, como freshdesk, que gerencia o email da Python Brasil). Dito isso, nada impede de vocês criarem uma conta onde for melhor para a Python Sul.

Outro ponto, eu vi que você preencheu o [formulário de solicitação de apoio da APyB](https://docs.google.com/forms/d/e/1FAIpQLSdtHD8t-P7dYqoCBonXOCv_WsHhmekZvulKOMHHJagxG1laSg/viewform) para Python Sul 2020, quando você puder, preenche de novo para termos os dados atualizados.
